x/ccv/provider/keeper/power_shaping_test.go

```diff
@@ -296,22 +301,22 @@ func TestCapValidatorSet(t *testing.T) {
 		},
 		{
 			name:                   "ValidatorSetCap = 2, with priority list",
-			powerShapingParameters: providertypes.PowerShapingParameters{ValidatorSetCap: 2, Prioritylist: []string{string(validatorA.ProviderConsAddr), string(validatorB.ProviderConsAddr)}},
+			powerShapingParameters: providertypes.PowerShapingParameters{ValidatorSetCap: 2, Prioritylist: []string{"cosmosvalcons1ezyrq65s3gshhx5585w6mpusq3xsj3ayzf4uv6", "cosmosvalcons1nx7n5uh0ztxsynn4sje6eyq2ud6rc6klc96w39"}},
```

Comment:
this seems nicer before, since it was more obvious what the addresses correspond to

Reply:
here and also below, could you pull these either from the validator structs, or reference the variables defined above otherwise?

Reply:
I’ve made some changes, let me know what you think.

Reply:
I think it looks better!